cnxepub/formatters.py

```diff
@@ -307,7 +317,7 @@ def __bytes__(self):
   </body>
 </html>
 """
-HTML_DOCUMENT = """\
+HTML_DOCUMENT = u"""\
```

Comment:
Yeah, was getting a spurious error from the template formatter that googling indicated was am encoding issue. Was actually caused by interactions with the debugger. Will remove.